examples/rapids/mlflow_project/README.md

```diff
@@ -0,0 +1,47 @@
+### Train and Publish Locally With RAPIDS and MLflow
+**[RAPIDS](https://rapids.ai/)** is a suite of open source libraries for GPU-accelerated analytics. 
+
+**[RAPIDS cuML](https://github.com/rapidsai/cuml)** matches the scikit-learn API, so it can build on MLflow's existing support for scikit-learn-like models to support
+persistence and deployment."
+
+The example workflows below train RAPIDs regression models to predict airline flight delays, using MLflow to log models and deploy them as local REST API endpoints for real-time inference.
+
+#### Jupyter Notebook Workflow
+[Jupyter Notebook](notebooks/rapids_mlflow.ipynb)
+
+#### CLI Based Workflow
+1. Download data
+    1. `cd examples/rapids/mlflow_project`
+        ```shell script
+        # Download the file
+        wget -N https://rapidsai-cloud-ml-sample-data.s3-us-west-2.amazonaws.com/airline_small.parquet
+        ```
+1. Set MLflow tracking uri
+    1. ```shell script
+        export MLFLOW_TRACKING_URI=sqlite:////tmp/mlflow-db.sqlite
+       ```
+1. Train the model using a single run.
+    1. ```shell script
+       # Launch the job
+       mlflow run . -b local -e hyperopt \
```

Comment:
We should be able to remove the `-b local` from this `mlflow run` invocation and the one below, as the default backend for project execution is "local"